src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewDataMigrationSelectedDB.cs

```diff
@@ -0,0 +1,122 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.DataMigration.Models;
+
+namespace Microsoft.Azure.Commands.DataMigration.Cmdlets
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmDataMigrationSelectedDB", DefaultParameterSetName = SqlServerSqlDbParameterSet), OutputType(typeof(MigrateSqlServerSqlDbDatabaseInput))]
```

Comment:
Since your module is not part of AzureRM at the moment, you could remove the previous cmdlet "New-AzureRmDataMigrationSqlServerSqlDbSelectedDB" and just add it as an alias to this new cmdlet.  This will be a breaking change, as the user will need to provide the new switchparameter to get the right parameter set, but I think the overall experience of having a small number of straightforward cmdlets will be much better in the long run. Let me know what you think.

Reply:
Makes sense, will remove it.

Reply:
In fact, I'll also not add it as alias to the new cmdlet. Anyways, even with the alias, you would need to add the switch parameter.

Reply:
Fixed.